src/mixins/jsonToCsv.js

```diff
@@ -0,0 +1,218 @@
+// TODO: use helper from ionicSdk instead of the mixin to convert the JSON to CSV
```

Comment:
This should be removed as we no longer send csv

Reply:
Ok sir

Reply:
This is removed by Yash sir in another MR